platform/iphone/camera_ios.mm

```diff
@@ -158,7 +158,7 @@ - (void)captureOutput:(AVCaptureOutput *)captureOutput didOutputSampleBuffer:(CM
 	} else if (dataCbCr == NULL) {
 		print_line("Couldn't access CbCr pixel buffer data");
 	} else {
-		UIDeviceOrientation orientation = [[UIApplication sharedApplication] statusBarOrientation];
+		UIInterfaceOrientation orientation = [[UIApplication sharedApplication] statusBarOrientation];
```

Comment:
Here the code already checked for `UIInterfaceOrientation` enum values further down, so that should be fine.